R/pivot-long.R

```diff
@@ -26,6 +26,25 @@
 #'   automatically guess from the data. Supply this argument when you want to
 #'   override that default. Should be a named list, where the names are
 #'   given by the value columns.
+#'
+#' @details
+#'
+#' `pivot_long` is a generalisation of the function `gather`, it differs from
```

Comment:
This feels a bit too long for `pivot_long()` - I think you can indicate that it supersedes `gather()`, and maybe include a brief note on translation `gather()` code to `pivot_long()`? (i.e. `key` is now `names_to` and `value` is `values_to`)

Reply:
Should we include translation notes in the @section and also in the `Examples`, as I believe many users simply scroll to the bottom of a documentation page?